tests/unit_tests/layout/test_layout_array.py

```diff
@@ -222,61 +264,98 @@ def test_converting_center_coordinates(layout_array_north_four_LST_instance):
     assert _lon.value == pytest.approx(-17.8920302)
 
     _east, _north, _ = layout._array_center.get_coordinates("utm")
-    assert _north.value == pytest.approx(3185067.28)
-    assert _east.value == pytest.approx(217609.23)
+    assert _north.value == pytest.approx(3185066.0)
```

Comment:
That is a good question. These values are kind of untraceable since they are put by hand in the tests. maybe we were using different EPSG coordinates before (or derived it differently) ? Do you know if these values are slightly different from the different production versions ?

Reply:
We should understand this. The EPSG zones are fixed for North and South and we don't expect differences here. There might have been a bug in the past, but let's try and understand what is happening.

Reply:
The latest info I am familiar with is in [this GitLab location](https://gitlab.cta-observatory.org/cta-science/simulations/simulation-and-performance-parallel-mode/layout-optimisation/layouts/-/tree/main/coordinates/array-centres). The values there are actually slightly different from the ones we have and I think we should update all of the files and tests we are using.

I can't find in the history of these files the original values we see here, but it doesn't matter I guess. They changed quite a few times so perhaps what we had is a snapshot of some time in the past. 

One thing is clear to me though, we shouldn't keep this info in so many files, it should be placed only in the DB and the files should just point to the site. I know @GernotMaier wants to keep this info in the file though, so maybe it should be a subject of discussion in our next call.

Reply:
But here we test simply the transformation from mercator to UTM and I would vote for keeping the numbers here. We test exactly what we want in the fastest and simplest possible way.

Reply:
I didn't mean to remove these numbers from here, I meant to remove them from the ecsv files we have in `data/layout/` or in the DB. I.e., files like [telescope_positions-North-1LST.ecsv](https://github.com/gammasim/gammasim-tools/blob/master/data/layout/telescope_positions-North-1LST.ecsv) or [telescope_positions-South-TestLayout.ecsv](https://github.com/gammasim/gammasim-tools/blob/master/data/layout/telescope_positions-South-TestLayout.ecsv)

Reply:
I agree we can discuss that too, I am adding this topic to our notes